src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/commands.py

```diff
@@ -49,5 +81,5 @@ def load_command_table(self, _):
 
     with self.command_group('eventgrid topic-type', topic_type_mgmt_util) as g:
         g.command('list', 'list')
-        g.show_command('show', 'get')
+        g.command('show', 'get')
```

Comment:
Should be `show_command`. Why was this changed?

Reply:
Fixed